ietf/static/js/ipr-search.js

```diff
@@ -1,8 +1,8 @@
 $(document)
     .ready(function () {
         // hack the "All States" check box
-        $("#id_state")
-            .addClass("list-inline");
+        $("#id_state .form-check")
+            .addClass("form-check-inline");
```

Comment:
Can this not be done in the template?

Reply:
These forms would need some serious work to better work with bs.